[2025-04-21T23:54:18.781+0000] {processor.py:186} INFO - Started process (PID=2849) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:18.785+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:54:18.793+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:18.787+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:19.024+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.024+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:54:19.026+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.026+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:54:19.031+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.031+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:54:19.031+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.031+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:54:19.573+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:19.711+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.710+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:project_historydata_ETL
[2025-04-21T23:54:19.718+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.718+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:project_historydata_ETL
[2025-04-21T23:54:19.721+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.721+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:project_historydata_ETL
[2025-04-21T23:54:19.724+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.723+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:project_historydata_ETL
[2025-04-21T23:54:19.726+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.726+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:project_historydata_ETL
[2025-04-21T23:54:19.728+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.728+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:project_historydata_ETL
[2025-04-21T23:54:19.730+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.729+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:project_historydata_ETL
[2025-04-21T23:54:19.730+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.730+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:54:19.736+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.736+0000] {dag.py:3252} INFO - Creating ORM DAG for project_historydata_ETL
[2025-04-21T23:54:19.742+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:19.742+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:54:19.759+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 1.002 seconds
[2025-04-21T23:54:50.286+0000] {processor.py:186} INFO - Started process (PID=2865) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:50.288+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:54:50.289+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.289+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:50.338+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.338+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:54:50.339+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.339+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:54:50.340+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.340+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:54:50.341+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.341+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:54:50.705+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:54:50.720+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.720+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:54:50.738+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:54:50.738+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:54:50.746+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 0.478 seconds
[2025-04-21T23:55:21.286+0000] {processor.py:186} INFO - Started process (PID=2881) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:21.289+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:55:21.290+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.290+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:21.334+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.334+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:55:21.335+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.335+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:55:21.336+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.336+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:55:21.336+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.336+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:55:21.929+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:21.935+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.935+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:55:21.954+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:21.953+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:55:21.961+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 0.688 seconds
[2025-04-21T23:55:52.443+0000] {processor.py:186} INFO - Started process (PID=2897) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:52.448+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:55:52.450+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.449+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:52.510+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.509+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:55:52.511+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.511+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:55:52.514+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.514+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:55:52.514+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.514+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:55:52.862+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:55:52.869+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.869+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:55:52.889+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:55:52.889+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:55:52.897+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 0.461 seconds
[2025-04-21T23:56:23.732+0000] {processor.py:186} INFO - Started process (PID=2913) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:23.738+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:56:23.742+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:23.740+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:23.842+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:23.840+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:56:23.846+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:23.846+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:56:23.848+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:23.848+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:56:23.849+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:23.849+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:56:24.269+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:24.282+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:24.281+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:56:24.301+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:24.301+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:56:24.319+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 0.609 seconds
[2025-04-21T23:56:54.790+0000] {processor.py:186} INFO - Started process (PID=2929) to work on /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:54.793+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/project_historydata_ETL.py for tasks to queue
[2025-04-21T23:56:54.795+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:54.795+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:54.848+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:54.848+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-21T23:56:54.850+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:54.849+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-04-21T23:56:54.850+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:54.850+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-21T23:56:54.851+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:54.851+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T23:56:55.258+0000] {processor.py:925} INFO - DAG(s) 'project_historydata_ETL' retrieved from /opt/airflow/dags/project_historydata_ETL.py
[2025-04-21T23:56:55.276+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:55.276+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-04-21T23:56:55.311+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:56:55.311+0000] {dag.py:4156} INFO - Setting next_dagrun for project_historydata_ETL to 2025-04-20 02:30:00+00:00, run_after=2025-04-21 02:30:00+00:00
[2025-04-21T23:56:55.332+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/project_historydata_ETL.py took 0.552 seconds
